### DISCUSSION TITLE
为啥更新到最新版本后，win10 开机多了一个 Mihomo 的网络？

### DISCUSSION THREAD

**Discussion by piwawa** [clash-verge-rev/clash-verge-rev#2197] [Q&A] [answered]
最近突然冒出来的，电脑就用 clash verge 和 v2ray

![image](https://github.com/user-attachments/assets/9eb18c82-0c05-4c6e-b580-1c90d066d8b2)

![image](https://github.com/user-attachments/assets/30ee74b7-11e2-45d0-a3d8-ffbc1a8f46b2)

**Comment by TomZhang1019** [comment, marked as answer]
设置 - Tun(虚拟网卡)模式 - Tun网卡名称

当你开启Tun模式时你电脑的流量会通过此网卡发送
